### PR TITLE
Update stream limits and connection limits

### PR DESCRIPTION
**Pull request by p-shahi**

### CODE REVIEW

**Review by maschad — APPROVED** [review]

File: packages/frontend/src/lib/libp2p.ts [hunk 1]
```diff
@@ -64,12 +70,15 @@ export async function startLibp2p() {
...
     identify: {
-      maxPushOutgoingStreams: 100,
-      maxInboundStreams: 100,
-      maxOutboundStreams: 100,
+      // these are set because we were seeing a lot of identify and identify push
+      // stream limits being hit
+      maxPushOutgoingStreams: 1000,
+      maxPushIncomingStreams: 1000,
+      maxInboundStreams: 1000,
+      maxOutboundStreams: 1000,
```
Comment (achingbrain) [inline packages/frontend/src/lib/libp2p.ts:78]: I am concerned by this.  These limits are per connection, this means you're allowing 1000 identify streams to be open at the same time on each connection.

If you're seeing errors it sounds like something's not cleaning up the streams.

Maybe related: https://github.com/libp2p/js-libp2p/pull/1424

File: packages/frontend/src/lib/libp2p.ts [hunk 2]
```diff
@@ -45,8 +51,8 @@ export async function startLibp2p() {
     }),],
     connectionEncryption: [noise()],
     connectionManager: {
-      maxConnections: 200,
-      minConnections: 1,
+      maxConnections: 10,
+      minConnections: 50,
     },
```
Comment (achingbrain) [inline packages/frontend/src/lib/libp2p.ts:56]: `minConnections` needs to be less than `maxConnections`.

Is there a reason you're overriding this?  If not I'd just remove the whole `connectionManager` block.

File: packages/frontend/src/lib/libp2p.ts (hunk 1 above)
Comment (achingbrain) [reply packages/frontend/src/lib/libp2p.ts:78]: Do what you have to to appease the demo gods though - this is more of a note to self to investigate.

File: packages/frontend/src/lib/libp2p.ts (hunk 2 above)
Comment (p-shahi) [reply packages/frontend/src/lib/libp2p.ts]: removed, I'm not sure why it was added. @maschad any idea?

File: packages/frontend/src/lib/libp2p.ts (hunk 1 above)
Comment (p-shahi) [reply packages/frontend/src/lib/libp2p.ts:74]: Yeah the upper bounds here are crazy, but we were hitting the limits quite

**Pull request merged**